slatedb/src/db/builder.rs

```diff
@@ -736,6 +737,15 @@ impl<P: Into<Path>> CompactorBuilder<P> {
         self
     }
 
+    /// Sets the compaction scheduler supplier to use for the compactor.
+    pub fn with_scheduler_supplier(
```

Comment:
Don't we already have this in `with_compaction_scheduler_supplier`?

Reply:
oh, no, we don't actually. This change is in `CompactionBuilder`, not `DbBuilder`. `DbBuilder` has a `with_compaction_scheduler_supplier`.  The `scheduler_supplier` in `CompactionBuilder` was hardcoded, and could not be modified.

Reply:
Ahh!